src/funding/card/config.jsx

```diff
@@ -90,85 +82,47 @@ export function getCardConfig() : FundingSourceConfig {
-            // Testing for Inline Guest black button
-            if (cardButtonExperiment && locale.lang === 'en' && locale.country === 'US' && !__TEST__) {
-                return (
-                    <div>
-                        <style
-                            nonce={ nonce }
-                            innerHTML={ `
-                                #black-credit-card-button {
-                                    background: #000;
-                                    color: #fff;
-                                    font-size: 1.15em;
-                                    border-radius: 4px !important;
-                                }
-                            ` } />
-                        <button
-                            type="button"
-                            id="black-credit-card-button"
-                            onClick={ event => onClick(event) }
-                            class={ `${ CLASS.BUTTON }` }
-                            { ...{
-                                [ ATTRIBUTE.FUNDING_SOURCE ]: FUNDING.CARD
-                            } }>
-                            Debit or Credit Card
-                        </button>
-                    </div>
-                );
-            }
-
-            let maxCards = 4;
+        logoColors: {
+            [ BUTTON_COLOR.WHITE  ]:  BUTTON_COLOR.BLACK,
+            [ DEFAULT ]:              BUTTON_COLOR.WHITE
+        },
 
-            // $FlowFixMe
-            if (maxCardForCountry[locale.country]) {
-                maxCards = maxCardForCountry[locale.country];
-            }
+        handleClick: true,
 
-            return CARD_PRIORITY.map(name => {
-
-                const cardEligibility = fundingEligibility[FUNDING.CARD];
-
-                if (!cardEligibility || !cardEligibility.vendors || !cardEligibility.vendors[name] || !cardEligibility.vendors[name].eligible) {
-                    return null;
-                }
-
-                const vendorConfig = vendors[name];
-
-                if (!vendorConfig) {
-                    return null;
-                }
-
-                const { Label } = vendorConfig;
-                
-                return (
-                    <div
-                        class={ `${ CLASS.CARD } ${ CLASS.CARD }-${ name }` }
-                        onClick={ event => onClick(event, { card: name }) }
-                        tabindex='0'
-                        role='button'
-                        { ...{
-                            [ ATTRIBUTE.FUNDING_SOURCE ]: FUNDING.CARD,
-                            [ ATTRIBUTE.CARD ]:           name
-                        } }
-                    >
-                        <Label />
-                    </div>
-                );
-            }).filter(Boolean).slice(0, maxCards);
+        Logo: ({ content, logoColor, locale = {} }) => {
+            const { lang } = locale;
+            const isRTL = isRTLLanguage(lang);
+            return (
+                <Fragment>
```

Comment:
@bluepnume do you know how to fix the flowtype error on this line
```ComponentFunctionType [1] is incompatible with string [2].```

https://travis-ci.org/paypal/paypal-checkout-components/builds/625986580#L664